redis-driver/pom.xml

```diff
@@ -66,13 +66,22 @@
         <maven.compile.sourceLevel>1.8</maven.compile.sourceLevel>
     </properties>
     <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
+    	 
+    	 <dependency>
+             <groupId>javax.json</groupId>
```

Comment:
Once these dependencies are in the parent we don't need to replace this:
https://github.com/eclipse/jnosql-diana-driver/blob/master/pom.xml#L91